### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/deployment/manually.md`

### PR DESCRIPTION
**Pull request by JoaoGustavoRogel**
Portuguese translation
- docs/pt/docs/deployment/manually.md

### CODE REVIEW

**Review by nazarepiedady — APPROVED** [review]

**Review by ceb10n — CHANGES_REQUESTED** [review]
Great work!! 🚀 🚀 🚀 

Just need to add the translation to the blocks `note` and `warning`

File: docs/pt/docs/deployment/manually.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+```console
+$ uvicorn main:app --host 0.0.0.0 --port 80
+
+<span style="color: green;">INFO</span>:     Uvicorn running on http://0.0.0.0:80 (Press CTRL+C to quit)
+```
+
+</div>
+
+/// note
```
[the remark this answers is not in the document]
Comment (JoaoGustavoRogel) [reply docs/pt/docs/deployment/manually.md]: Thanks for the review! I've just added the fix in: https://github.com/fastapi/fastapi/pull/12210/commits/8780796f0ac3ed78ca61e602a16e49b359d017fb

File: docs/pt/docs/deployment/manually.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+```Python
+from main import app
+```
+
+///
+
+Cada programa de servidor ASGI alternativo teria um comando semelhante, você pode ler mais na documentação respectiva.
+
+/// warning
```
[the remark this answers is not in the document]
Comment (JoaoGustavoRogel) [reply docs/pt/docs/deployment/manually.md]: Added in: https://github.com/fastapi/fastapi/pull/12210/commits/8780796f0ac3ed78ca61e602a16e49b359d017fb

File: docs/pt/docs/deployment/manually.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+$ pip install "uvicorn[standard]"
+
+---> 100%
+```
+
+</div>
+
+Um processo semelhante se aplicaria a qualquer outro programa de servidor ASGI.
+
+/// tip | Dica
```
Comment (ceb10n) [inline docs/pt/docs/deployment/manually.md:105]: You just missed the quotes:

/// tip | ***"*Dica**"**

File: docs/pt/docs/deployment/manually.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+
+```Python
+from main import app
+```
+
+///
+
+Cada programa de servidor ASGI alternativo teria um comando semelhante, você pode ler mais na documentação respectiva.
+
+/// warning | Aviso
```
Comment (ceb10n) [inline docs/pt/docs/deployment/manually.md:146]: You just missed the quotes:

/// warning | ***"*Aviso**"**

**Review by ceb10n — CHANGES_REQUESTED** [review]
Thank you João Gustavo! ✨ 

You just missed the quotes in the translations

File: docs/pt/docs/deployment/manually.md [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+
+```console
+$ uvicorn main:app --host 0.0.0.0 --port 80
+
+<span style="color: green;">INFO</span>:     Uvicorn running on http://0.0.0.0:80 (Press CTRL+C to quit)
+```
+
+</div>
+
+/// note | Nota
```
[the remark this answers is not in the document]
Comment (JoaoGustavoRogel) [reply docs/pt/docs/deployment/manually.md]: Fixed in: https://github.com/fastapi/fastapi/pull/12210/commits/99ce96f62e6f1780b2ed94d7e809966c15c9ae1d

File: docs/pt/docs/deployment/manually.md (hunk 4 above)
Comment (JoaoGustavoRogel) [reply docs/pt/docs/deployment/manually.md]: Fixed in: https://github.com/fastapi/fastapi/pull/12210/commits/99ce96f62e6f1780b2ed94d7e809966c15c9ae1d

File: docs/pt/docs/deployment/manually.md (hunk 3 above)
Comment (ceb10n) [reply docs/pt/docs/deployment/manually.md]: Tha's right, they are mandatory.

Thank you for the fix João! ✨

**Review by ceb10n — APPROVED** [review]
❤️

**Review by marcelomarkus — APPROVED** [review]

**Comment by alejsdev** [comment]
Thanks for your time and effort contributing to the project! @JoaoGustavoRogel @marcelomarkus @nazarepiedady @ceb10n :rocket: :sparkles:

**Pull request merged**